docs/guide/snaps.md

```diff
@@ -76,7 +76,9 @@ Once you've connected to Flask, smash that `Send Hello` button, and watch the sn
 
 ## Where to Go from Here
 
-- If you're interested in developing your own snap, your next destination is probably [the snaps development guide](./snaps-development-guide.html).
-- You may also wish to check out the first functional key management snap ever made, [the Filecoin Snap, or `filsnap`](https://github.com/Chainsafe/filsnap/).
-- Check out the list of [Awesome MetaMask Snaps](https://github.com/piotr-roslaniec/awesome-metamask-snaps) and other resources.
```

Comment:
Do we not want to link to the snaps list anymore?

Reply:
Not from here.